packages/calculator/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@pie-ui/pie-calculator",
+  "version": "1.0.0",
+  "description": "A pie calculator component",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "./node_modules/.bin/mocha  --require test/setup test/**/*-test.js"
+  },
+  "author": "harry",
+  "license": "ISC",
+  "dependencies": {
+    "@pie-framework/material-ui-calculator": "^1.1.1",
+    "material-ui": "^1.0.0-beta.27",
+    "material-ui-icons": "^1.0.0-beta.17",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1",
+    "react-draggable": "^3.0.5"
+  },
+  "devDependencies": {
+    "babel": "^6.23.0",
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-preset-airbnb": "^2.4.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-react-hmre": "^1.1.1",
+    "babel-preset-stage-0": "^6.24.1",
+    "babel-register": "^6.24.1",
+    "enzyme": "^3.3.0",
+    "expect": "^22.4.0",
+    "jsdom": "^11.6.2",
```

Comment:
I think some of these dependencies can be removed? eg: jsdom, react-svg-loader etc?